FordLabs/PeopleMover
# Update role and notes font styles

## Issue
Resolves #535 

## What was done
- [x] Updated the reassigned person role text to be regular font instead of italics
- [x] Updated the person and product notes form text to inherit the font used throughout the rest of the app

## How to test
1. Assign a person with a role to a product and ensure that the role text that shows up in the reassigned drawer is no longer italicized.
2. Create a product and person with notes and ensure that the notes text matches the font of the other input fields.

### FOR DEVS
Feature URL: https://featurebranchui.apps.pd01i.edc1.cf.ford.com/

Feature Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20FeatureBranch/

### FOR APPROVAL
Approval URL: https://approvalbranchui.apps.pd01i.edc1.cf.ford.com/

Approval Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20ApprovalBranch/
